src/kit.ts

```diff
@@ -989,12 +989,9 @@ export async function effectiveKitEnvironment(kit: Kit, opts?: expand.ExpansionO
                 path_list.push(mingwPath);
             }
             if (env.hasOwnProperty('PATH')) {
-                let current_path_list = (env['PATH']?.split(';') ?? []);
-                if (mingwPath) {
-                    // Remove other mingw from PATH as it will cause conflict
-                    current_path_list = current_path_list.filter(p => !isMingw(p));
-                }
-                path_list.unshift(current_path_list.join(';'));
+                const current_path_list = (env['PATH']?.split(';') ?? []);
```

Comment:
Given that we aren't filtering `current_path_list` anymore, we shouldn't need to `split()` and `join()`. We can just directly push `env['PATH']` to `path_list`.

Reply:
fixed